schema/fair.js

```diff
@@ -0,0 +1,60 @@
+import gravity from '../lib/loaders/gravity';
+import cached from './fields/cached';
+import Profile from './profile';
+import {
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLBoolean,
+  GraphQLNonNull
+} from 'graphql';
+
+let FairType = new GraphQLObjectType({
+  name: 'Fair',
+  fields: () => ({
+    cached: cached,
+    id: {
+      type: GraphQLString
+    },
+    profile: {
+      type: Profile.type,
+      resolve: ({ default_profile_id, organizer }) => {
+        let id = default_profile_id || organizer && organizer.profile_id;
+        return gravity(`profile/${id}`)
+      }
+    },
+    has_full_feature: {
+      type: GraphQLBoolean,
+      resolve: ({ has_full_feature }) => has_full_feature
```

Comment:
So you don't actually have to pull out these attributes with resolves, only resolve if you need to compute a value or there's something async